docs/content/steps/saving-steps.md

```diff
@@ -6,9 +6,9 @@ slug: /concepts/saving-steps
 
 :::danger
 
-Workflow Steps from Apps are a deprecated feature.
+Steps from apps are a deprecated feature.
 
-Workflow Steps from Apps are different than, and not interchangable with, Slack automation workflows. We encourage those who are currently publishing Workflow Steps from Apps to consider the new [Slack automation features](https://api.slack.com/automation), such as [custom functions for Bolt](/concepts/creating-custom-functions).
+Steps from apps are different than, and not interchangable with, Slack automation workflows. We encourage those who are currently publishing steps from apps to consider the new [Slack automation features](https://api.slack.com/automation), such as [custom steps for Bolt](/concepts/creating-custom-functions).
```

Comment:
```suggestion
Steps from apps are different than, and not interchangable with, Slack automation workflows. We encourage those who are currently publishing steps from apps to consider the new [Slack automation features](https://api.slack.com/automation), such as [custom steps for Bolt](/concepts/custom-steps).
```

Reply:
Good catch `/concepts/custom-step` is not out yet I will link to https://api.slack.com/automation/functions/custom-bolt instead